themes/app/src/scss/components/_accommodation.scss

```diff
@@ -0,0 +1,121 @@
+.accommodation__desc {
```

Comment:
When designing block components on your HomePage I have just noticed that we should control the spacing between each component using `padding` instead of `margin`.

<img width="453" alt="screen shot 2018-12-06 at 11 02 28 am" src="https://user-images.githubusercontent.com/27745093/49547094-aec56880-f946-11e8-9c7f-ea4895759e81.png">

Here you can see how the margin pushes the block below down. It would be nicer if we controlled the spacing within the block itself. So what I would do is then remove the `margin-bottom: 20px` from `.banner` and increase the `padding top/bottom` on the `.accomodation` class :) 

If that doesn't make sense come grab me and I can show ya

Reply:
<img width="320" alt="screen shot 2018-12-06 at 11 10 31 am" src="https://user-images.githubusercontent.com/27745093/49547453-973aaf80-f947-11e8-9919-58289ea45d5b.png">

I thhink we can add more `padding-bottom` on this area here :)